bridges/SteamBridge.php

```diff
@@ -4,59 +4,14 @@ class SteamBridge extends BridgeAbstract {
 	const NAME = 'Steam Bridge';
 	const URI = 'https://store.steampowered.com/';
 	const CACHE_TIMEOUT = 3600; // 1h
-	const DESCRIPTION = 'Returns games list';
+	const DESCRIPTION = 'Returns apps list';
 	const MAINTAINER = 'jacknumber';
 	const PARAMETERS = array(
 		'Wishlist' => array(
 			'username' => array(
 				'name' => 'Username',
 				'required' => true,
 			),
-			'currency' => array(
```

Comment:
Why removing currency ? It is still supported

Reply:
My bad, when you are login on Steam the currency doesn't apply. Tested on private mode and yes, currency url parameter still working. I'm fixing it.